src/main/java/seedu/address/model/client/Transaction.java

```diff
@@ -40,6 +42,19 @@ public static boolean isValidAmount(String test) {
         return test.matches(VALIDATION_REGEX);
     }
 
+    /**
+     * Throws a {@code ParseException} if the amount is out of range.
+     */
+    public static void verifyAmountIsWithinRange(double amount) throws ParseException {
+        double minAmount = -Math.pow(10, 9);
+        double maxAmount = Math.pow(10, 9);
+
+        if (amount < minAmount || amount > maxAmount) {
+            throw new ParseException(MESSAGE_AMOUNT_OUT_OF_RANGE);
+        }
+
+    }
```

Comment:
I feel like this method can be put in `ParserUtil`

Reply:
Ok I moved the method over